app/src/services/account-management.service.ts

```diff
@@ -376,7 +402,7 @@ export class UserLoginService {
       // first, load the valid tokens cached in the local store, if they are available
       // see: https://github.com/aws/amazon-cognito-identity-js/issues/71
       let cognitoUser = CognitoUtil.getCognitoUser();
-      cognitoUser.getSession( (err: Error, session: any) => {
+      cognitoUser.getSession((err: Error, session: any) => {
```

Comment:
I haven't addressed this comment. However, the [issue] you're citing in the code is now closed. 

[issue]:https://github.com/aws/amazon-cognito-identity-js/issues/71